assets/ts/stop/components/DepartureCard.tsx

```diff
@@ -76,33 +107,43 @@ const DepartureCard = ({
         {renderSvg("c-svg__icon", routeToModeIcon(route), true)}{" "}
         {routeName(route)}
       </a>
-      {sortedRoutePatternsByHeadsign.map(
+      {Object.entries(departures).map(
         ([
           headsign,
-          { direction_id: directionId, route_patterns: routePatterns }
+          departures
         ]) => {
+          let alerts : Alert[] = [];
+          let directionId = 0;
```

Comment:
📓 This is the only thing I am wondering about. Does this matter if there are no actual departures?